.github/workflows/client-release.yml

```diff
@@ -27,9 +32,20 @@ jobs:
       run: |
         echo "RELEASE_VERSION=$(git tag --sort=committerdate | grep -E '^v[0-9]' | tail -1 )" >> $GITHUB_ENV
         make version
-    - name: Build and Push Artifacts to gcloud
+
+    - uses: actions-rs/toolchain@v1
+      with:
+        toolchain: stable
+
+    - name: 🔨 Build
+      uses: actions-rs/cargo@v1
+      with:
+        command: build
+        args: --release --bins
```

Comment:
This also won't respect the toolchain.toml, meaning that tests are not run vs the same version of rustc as the final build